packages/amplication-client/src/Resource/git/dialogs/GitCreateRepo/GitCreateRepo.scss

```diff
@@ -42,4 +42,13 @@
     margin-bottom: var(--icon-spacing);
     font-size: 10px;
   }
+
+  &__info {
+    font-size: 10px;
```

Comment:
dont use fixed numbers, instead use variable when needed.
In this case, it is better to use the <Text component with the relevant textStyle and textColor properties.
see this for example
https://github.com/amplication/amplication/blob/12f27b00c2a05b3f5456439a235c9080ae5a80c1/packages/amplication-client/src/Resource/OverviewSecondaryTile/OverviewSecondaryTile.tsx#L35C22-L35C22